source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -225,4 +225,30 @@ execute a find operation on a collection of ``RecordTree`` types:
    .. output::
       :language: sh
 
-      RecordTree[content=Ranginui, left=RecordTree[content=..., left=RecordTree[content=..., right=RecordTree[content=Ikatere...]]
\ No newline at end of file
+      RecordTree[content=Ranginui, left=RecordTree[content=..., left=RecordTree[content=..., right=RecordTree[content=Ikatere...]]
+
+.. _fundamentals-records-get-codec:
+
+Retrieve the Record Codec
+-------------------------
+
+When working with records, consider using the ``RecordCodecProvider`` to get the
+record codec for Java records. For more information about codecs and why you might want
+to use them, see :ref:`Codecs <fundamentals-codecs>`. In summary, you use codecs
+to encode and decode Java objects to and from corresponding BSON types while
+minimizing duplicate code.
+
+.. tip::
+
+   When using the record codec, the annotations listed under
+   :ref:`Specify Component Conversion Using Annotations <fundamentals-records-annotations>`
+   are only supported on the record component.
```

Comment:
S: avoid passive voice
```suggestion
   The driver supports the annotations listed under
   :ref:`Specify Component Conversion Using Annotations <fundamentals-records-annotations>`
   only on the record component.
```

Also, what does "record component" mean?

Reply:
I moved this to be in the section specifically about annotations because I think it fits much better there. Otherwise this information is scattered in a couple of places.